python/pickles-in-pandas.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+- id: pickles-in-pandas
+  patterns:
+    - pattern-either:
+      - pattern: pandas.read_pickle(...)
+      - pattern: $DF.to_pickle(...)
```

Comment:
Add pattern: `x = $DF.to_pickle; x(...)`

Reply:
Also add testcase for:
```python
def foo(..., $X=$DF.to_pickle, ...):
    ...
    $X(...)
    ...
```

Reply:
I added two new patterns as that test case wouldn't have been caught by the original rule.